src/index.ts

```diff
@@ -1 +1,6 @@
 export { openpty } from "./pty";
+export { TtyRequest } from './client-server/termiosData';
+export { TtyClient } from "./client-server/ttyClient";
+export { TtyServer } from "./client-server/ttyServer";
```

Comment:
I think `TtyRequest`, `TtyClient`, and `TtyServer` are not included in the package. So I don't think their type definitions are needed either.